packages/shopify-codemod/test/transforms/convert-default-export-objects-to-named-exports.test.js

```diff
@@ -0,0 +1,11 @@
+import 'test-helper';
+import convertDefaultExportObjectsToNamedExports from 'convert-default-export-objects-to-named-exports';
+
+describe('convertDefaultExportObjectsToNamedExports', () => {
+  it('converts export object literals', () => {
+    expect(convertDefaultExportObjectsToNamedExports).to.transform('convert-default-export-objects-to-named-exports/literals');
+  });
+  it('converts export object variables', () => {
```

Comment:
I know this is so knitpicky, but I always prefer a space between calls like this :)